docs/es/week03/03.md

```diff
@@ -35,4 +35,4 @@ Properties of natural signals that are most relevant to CNNs are discussed in mo
 
 ## Práctica
 
- Discutiremos las propiedades mas relevantes de las señales (composicionalidad, estacionariedad, y localidad) en el contexto de CNNs . Explicaremos cómo el kérnel aprovecha estas propiedades utilizando las cualidades de esparcidad (sparcity), reutilización de parámetros (weight sharing), y el apilamiento (stacking) de capas convolucionales, al igual que las técnicas de amortiguación (padding) y  agrupamiento (pooling). Por último demostraremos una comparación entre una red densa (fully connected network) y una CNN para distintos datos.
+ Discutimos a detalle las propiedades de las señales naturales más relevantes para las CNN: composicionalidad, estacionariedad y localidad. Exploramos precisamente cómo es que un *kernel* aprovecha estas características utilizando las cualidades de esparcidad (*sparcity*), reutilización de parámetros (*weight sharing*) y el apilamiento (*stacking*) de capas convolucionales, al igual que las técnicas de relleno (*padding*) y muestreo (*pooling*). Por último, realizamos una comparación de desempeño entre una red densa (*fully connected network*) y una CNN para distintas modalidades de datos.

```

Comment:
suggest: "...las señales naturales más relevantes para las CNN: **composición, estacionaridad y localidad**.."

Reply:
I would prefer **composicionalidad** since it is the quality of being able to be composed and that's the same level in the other elements being coordinated.